weave/trace_server/interface/base_object_classes/human_annotation_column.py

```diff
@@ -0,0 +1,16 @@
+from typing import Optional
+
+from weave.trace_server.interface.base_object_classes import base_object_def
+
+
+class HumanAnnotationColumn(base_object_def.BaseObject):
+    json_schema: dict
+
+    # If true, all unique creators will have their
+    # own value for this feedback type. Otherwise,
+    # by default, the value is shared and can be edited.
+    unique_among_creators: bool = False
+
+    # If provided, this feedback type will only be shown
+    # when a call is generated from the given op
+    op_scope: Optional[list[str]] = None
```

Comment:
is this expected to be a ref string or op name?

Reply:
I think we want ref strings? like the frontend format `"weave:///griffin_wb/prod-evals-aug/op/Evaluation.summarize:*"`

Reply:
i see in your tests that this is just the name itself. you might consider using a ref here. then if you wanted to match any version you can use `weave:///e/p/name:*` (just like we support with `op_name` in the call filter

Reply:
oh, you commented before i finished reviewing. Looks like we are on the same page

Reply:
haha yeah okay fixing